src/ui/digital-exchange/components/list/ComponentList.js

```diff
@@ -53,6 +113,7 @@ ComponentList.defaultProps = {
   onWillMount: () => {},
   loading: false,
   digitalExchangeComponents: [],
+  listViewMode: 'gird-view',
```

Comment:
please extract grid-view to a constant: it's used in more than one file (here `gird-view` should be a typo, I guess